docs/ruleset_functions/get_zone_target_baseline_system.md

```diff
@@ -0,0 +1,97 @@
+		- change the system type for any zone in this bat: `for zone in zones_and_systems:`
+			- check if this zone is in the list of zone_ids associated with this bat: `if zone.id in list_building_area_types_and_zones[bat]["ZONE_IDS"]:`
+				- change the system type to: [secondary_system_type[0], "G3_1_1b"] - remember that expected_system_type_from_Table_G3_1_1 gives both the expected system type and a string explaining how that system type was chosen.  For the rule testing G3_1_1b, it is only asking whether the system was chosen from Table G3.1.1, not exactly how it got there: `zones_and_systems[zone] = [secondary_system_type[0], "G3_1_1b"]`
+
+- G3.1.1c "If the baseline HVAC system type is 5, 6, 7, 8, 9, 10, 11, 12, or 13 use separate single-zone systems conforming with the requirements of system 3 or system 4 for any HVAC zones that have occupancy, internal gains, or schedules that differ significantly from the rest of the HVAC zones served by the system. The total peak internal gains that differ by 10 Btu/h·ft2 or more from the average of other HVAC zones served by the system, or schedules that differ by more than 40 equivalent full-load hours per week from other spaces served by the system, are considered to differ significantly. Examples where this exception may be applicable include but are not limited to natatoriums and continually occupied security areas. This exception does not apply to computer rooms."
+- loop through the zones_and_systems to see if any of the zones meets Ge.1.1.c: `for zone in zones_and_systems:`
+	- use the function does_zone_meet_G3_1_1c to determine if this zone meets this requirement: `if does_zone_meet_G3_1_1c(B_RMI,zone,zones_and_systems[zone][0] == YES):`
+		- the zone meets the G3_1_1c requirements, choose system 3 or 4 based on climate zone: `if is_CZ_0_to_3a(climate_zone):`
+			- set the system to "SYS-3" and source to "G3_1_1c": `zones_and_systems[zone] = ["SYS-3",zones_and_systems[zone][0] + " G3_1_1c"]`
+		- else if the climate zone is_CZ_3b_3c_and_4_to_8: `elsif is_CZ_3b_3c_and_4_to_8(climate_zone):`
+			- set the system to "SYS-4" and source to "G3_1_1c": `zones_and_systems[zone] = ["SYS-4",zones_and_systems[zone][0] + " G3_1_1c"]`
+				
+- G3.1.1d "For laboratory spaces in a building having a total laboratory exhaust rate greater than 15,000 cfm, use a single system of type 5 or 7 serving only those spaces.  The lab exhaust fan shall be modeled as constant horsepower reflecting constantvolume stack discharge with outdoor air bypass."
+- not sure how to apply this one
+
+
+- G3.1.1e "Thermal zones designed with heating-only systems in the proposed design serving storage rooms, stairwells, vestibules, electrical/mechanical rooms, and restrooms not exhausting or transferring air from mechanically cooled thermal zones in the proposed design shall use system type 9 or 10 in the baseline building design."
```

Comment:
I think the first layer of logic is to identify whether the zone is heating-only. Is it in the `zone_meets_G3_1_1e` function?

Reply:
yes, it's in the `zone_meets_G3_1_1e` function